domain-ee/ee-ep-merge-app/src/python_src/service/ep_merge_machine.py

```diff
@@ -13,14 +13,17 @@
 from statemachine import State, StateMachine
 from util.contentions_util import ContentionsUtil, MergeException
 
-CANCELLATION_REASON = "Issues moved to pending EP Claim ID #%d"
+CANCEL_TRACKING_EP = "60"
```

Comment:
In the two places this gets used, it replaces the value `"65"` not `"60"`. Intentional?

Reply:
60 is right!